config/packages/prod/sulu_document_manager.yaml

```diff
@@ -1,49 +0,0 @@
-sulu_document_manager:
-    sessions:
-        default:
-            backend:
-                caches:
-                    meta: doctrine_phpcr.meta_cache_provider
```

Comment:
Can't we at least cache the metadata? Or does that also need to be cleared, when a new page is published?

Reply:
Metadata cache can be keeped 👍